CMSgov/easi-app
# [EASI-4129] Redis caching

# EASI-4129, EASI-4130

## Changes and Description

- Adds Redis to docker-compose local workflow.
- Uses Redis to cache system summary and roles CEDAR requests.

## How to test this change

Connect to the CMS VPN, start the app using docker-compose as normal, and make sure to set .envrc.local to allow connecting to CEDAR. System summary and roles requests should cache. Adding/removing users/roles to systems has cache invalidation applied as an example. No other CEDAR requests are cached in this draft.

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
